cli/harness/src/main/java/io/helidon/build/cli/harness/CommandParser.java

```diff
@@ -120,11 +123,12 @@ static CommandParser create(String... args) {
         return new CommandParser(argsList, commandName, new Resolver(params, properties), error);
     }
 
-    static String[] preProcessArgs(String[] args) {
+    static String[] preProcessArgs(Map<String, Parameter> params, String[] args) {
         List<String> result = new ArrayList<>();
         for (int i = 0; i < args.length; i++) {
-            if ("--args-file".equals(args[i]) && (i < args.length - 1)) {
+            if (ARGS_FILE_OPTION_ARGUMENT.equals(args[i]) && (i < args.length - 1)) {
```

Comment:
Actually, `--args-file` is a pre-processing argument, it does not make sense to expose to expose in global options.